test.py

```diff
@@ -0,0 +1,84 @@
+# vstart cluster me running before running this test.
```

Comment:
We can't leave this test script sitting in the root directory. ;) Put it in one of the appropriate test folders.

Reply:
Maybe we can just add the test in `src/test/libcephfs/access.cc`.